### PR TITLE
175851 docs linting

### PR DESCRIPTION
**Pull request by adunndevster**

### CODE REVIEW

File: docs-linting/src/extension.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+/**
+ * Method to check for the docs custom markdownlint value.
+ * Checks for markdownlint.customRules property.  If markdownlint isn't installed, do nothing.  If markdownlint is installed, check for custom property values.
+ */
+export function checkMarkdownlintCustomProperty() {
+    const { msTimeValue } = generateTimestamp();
+    const customProperty = "markdownlint.customRules";
+    const customRuleset = "{docsmsft.docs-linting}/markdownlint-custom-rules/rules.js";
```
Comment (jamarw) [inline docs-linting/src/extension.ts]: Hi @adunndevster , do you think we should add logic to delete the custom ruleset entry for docs-markdown as well?  I don't think it's got any negative impact aside from noise in the output window.  Thanks

Comment (adunndevster) [reply docs-linting/src/extension.ts]: Ah! That's what was causing that. @jamarw - you are a wizard.

Comment (adunndevster) [reply docs-linting/src/extension.ts]: @jamarw - Okay I've updated this. The first time you run docs-linting you will see a little mess... it tries to load the old rules, is unable to, then refreshes the cache, and loads fine. After that the messages are clean up. Is this okay?

**Comment by bharney0** [comment]
#212

File: docs-linting/src/extension.ts (hunk 1 above)
Comment (jamarw) [reply docs-linting/src/extension.ts]: @adunndevster Looks great!

**Pull request merged**